addons/im_livechat/static/tests/qunit_suite_tests/components/discuss_tests.js

```diff
@@ -366,5 +375,65 @@ QUnit.test('call buttons should not be present on livechat', async function (ass
     );
 });
 
+QUnit.test('reaction button should not be present on livechat', async function (assert) {
+    assert.expect(1);
+
+    const pyEnv = await startServer();
+    const mailChannelId1 = pyEnv['mail.channel'].create({
+        anonymous_name: "Visitor 11",
+        channel_type: 'livechat',
+        livechat_operator_id: pyEnv.currentPartnerId,
+        channel_partner_ids: [pyEnv.currentPartnerId, pyEnv.publicPartnerId],
+    });
+    const { click } = await this.start({
+        discuss: {
+            params: {
+                default_active_id: `mail.channel_${mailChannelId1}`,
+            },
+        },
+    });
+    await afterNextRender(() => {
+        document.querySelector(`.o_ComposerTextInput_textarea`).focus();
+        document.execCommand('insertText', false, "Test");
+    });
```

Comment:
`await insertText('.o_ComposerTextInput_textarea', "Test");`
(get `insertText` from `start` like you did with `click`)